File: list_files.py

import os

json_folder = os.path.join(os.path.dirname(__file__), 'json')
prefix = 'https://datosabiertos.gob.pe/dataset/'

for filename in os.listdir(json_folder):
    if filename.endswith('.json'):
        filename_without_ext = os.path.splitext(filename)[0]
        print(prefix + filename_without_ext)

File: listFiles.js

const fs = require('fs');
const path = require('path');

const jsonFolder = path.join(__dirname, 'json');
const prefix = 'https://datosabiertos.gob.pe/dataset/';

fs.readdir(jsonFolder, (err, files) => {
    if (err) {
        return console.error('Unable to scan directory: ' + err);
    }
    files.forEach(file => {
        if (path.extname(file) === '.json') {
            const filenameWithoutExt = path.basename(file, '.json');
            console.log(prefix + filenameWithoutExt);
        }
    });
});